src/lib/openapi/spec/health-overview-schema.ts

```diff
@@ -14,16 +14,23 @@ export const healthOverviewSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['version', 'name'],
+    description: `An overview of a project and its health as described in the documentation on [technical debt](https://docs.getunleash.io/reference/technical-debt)`,
     properties: {
         version: {
             type: 'number',
+            description: 'The project overview version.',
```

Comment:
For clarity: is this the version of the format used for this object? If so, maybe we could make it a little clearer. But also: wouldn't changing the format require a new schema? If so, maybe this should be:
 
```suggestion
            description: 'The project overview version.',
            enum: [1],
```

Reply:
This is a good question. So if Unleash always returns only a single schema, then we wouldn't need to delete it, we could just evolve it between versions and update the version from `[1]` to `[2]`, for instance.

If we want to return multiple schemas, we'd have to keep them around. 

am I making sense? did that answer your question?

Reply:
The interesting part is that changing this to enum: [1] introduces build errors as it's being passed around as a 'number' in the domain